OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -28,6 +28,18 @@ public CaptureActor(Actor self, Target target)
 			manager = self.Trait<CaptureManager>();
 		}
 
+		protected override void TickInner(Actor self, Target target, bool targetIsDeadOrHiddenActor)
+		{
+			if (target.Type != TargetType.Invalid && enterActor != target.Actor)
```

Comment:
Sorry, I should have worded the request clearer: We need to check if the `Type` is `TargetType.Actor`, since `target.Actor` is null in the other cases. (This crashes currently when targeting frozen actors.)

Reply:
Fixed